### PR TITLE
doc: Fix Matter bridge application overview

### PR DESCRIPTION
**Pull request by markaj-nordic**
Added missing bits:
* List of supported bridged devices
* Clarification of the possible extensions of the app

### CODE REVIEW

File: applications/matter_bridge/README.rst [hunk 1]
```diff
@@ -9,6 +9,13 @@ The devices on the non-Matter side of the Matter bridge are called *bridged devi
 
 The Matter bridge device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.11ax (Wi-Fi 6) network.
 
+Currently the Matter bridge application supports the following types of *bridged devices*:
+* Bluetooth® LE peripheral devices - devices that operate on real data (e.g. sensor measurements, lighting state)
```
Comment (wiba-nordic) [inline applications/matter_bridge/README.rst:13]:
```suggestion
Currently the Matter bridge application supports the following types of *bridged devices*:

* Bluetooth® LE peripheral devices - Devices that operate on real data (such as sensor measurements or lighting state).
```

File: applications/matter_bridge/README.rst [hunk 2]
```diff
@@ -9,6 +9,13 @@ The devices on the non-Matter side of the Matter bridge are called *bridged devi
...
+* Simulated devices - devices that interact with the bridge by using data fabricated programmatically by the software
```
Comment (wiba-nordic) [inline applications/matter_bridge/README.rst:14]:
```suggestion
* Simulated devices - Devices that interact with the bridge by using data fabricated programmatically by the software.
```

File: applications/matter_bridge/README.rst [hunk 3]
```diff
@@ -9,6 +9,13 @@ The devices on the non-Matter side of the Matter bridge are called *bridged devi
...
+
+Note that, apart from already implemented *bridged devices*, the Matter bridge architecture allows adding support for other devices that run over diverse connectivity technologies (e.g. Zigbee, Bluetooth® Mesh).
```
Comment (wiba-nordic) [inline applications/matter_bridge/README.rst:16]:
```suggestion
Note that, in addition to already implemented *bridged devices*, the Matter bridge architecture allows you to add support for other devices that run over diverse connectivity technologies, such as Zigbee or Bluetooth® Mesh.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]
Thank you ❤️

**Pull request merged**